packages/test/bin/tfm-publish-coverage.js

```diff
@@ -4,10 +4,10 @@
 const cwd = process.cwd();
 const { exec } = require('child_process');
 
-const coverallsPath = require.resolve('coveralls');
```

Comment:
`require.resolve('coveralls')` returns `coveralls/index,js` while we need `./.bin/coveralls`